### PR TITLE
[DOC] Index construction: hint regarding disk/space quota

### PR DESCRIPTION
**Pull request by cpockrandt**
As suggested in #2083

### CODE REVIEW

File: manual/source/Tutorial/DataStructures/Indices/StringIndices.rst [hunk 1]
```diff
@@ -47,6 +47,14 @@ We will now show how we can create the different indices in SeqAn before we show
 All the mentioned indices belong to the generic :dox:`Index` class.
 A SeqAn index needs two pieces of information: the type of the :dox:`String` or :dox:`StringSet` to be indexed and the index specialization, such as :dox:`IndexEsa` or :dox:`FMIndex`.
 
+.. important::
+
+    Indices based on suffix arrays (also including the FM index) are built using secondary memory. Building large indices can throw errors when you are running out of disk space or quota. In that case you can change the path of your tmpfs used for constructing the index.
```
Comment (h-2) [inline manual/source/Tutorial/DataStructures/Indices/StringIndices.rst]: 1. Please adhere to maximum line width, also introduce line breaks after full stop.
2. The third sentence is confusing. Not everyone has a tmpFS and if they have, you still cannot change it's path. I would propose: `The directory used for temporary storage can be changed by specifying the TMPDIR environment variable:`

File: manual/source/Tutorial/DataStructures/Indices/StringIndices.rst [hunk 2]
```diff
@@ -47,6 +47,14 @@ We will now show how we can create the different indices in SeqAn before we show
...
+
+    .. code-block:: console
+
+       # export TMPDIR=/somewhere/else/with/more/space
```
Comment (h-2) [inline manual/source/Tutorial/DataStructures/Indices/StringIndices.rst]: Does this also work on windows? If not, please add similar instructions!

File: manual/source/Tutorial/DataStructures/Indices/StringIndices.rst [hunk 3]
```diff
@@ -47,6 +47,21 @@ We will now show how we can create the different indices in SeqAn before we show
 All the mentioned indices belong to the generic :dox:`Index` class.
 A SeqAn index needs two pieces of information: the type of the :dox:`String` or :dox:`StringSet` to be indexed and the index specialization, such as :dox:`IndexEsa` or :dox:`FMIndex`.
 
+.. important::
+
+    Indices based on suffix arrays (also including the FM index) are built using secondary memory.
+    Building large indices can throw errors when you are running out of disk space or quota.
```
Comment (smehringer) [inline manual/source/Tutorial/DataStructures/Indices/StringIndices.rst]: Building large indices therefore can throw errors `,` when you are running... ? (comma)

File: manual/source/Tutorial/DataStructures/Indices/StringIndices.rst [hunk 4]
```diff
@@ -47,6 +47,21 @@ We will now show how we can create the different indices in SeqAn before we show
...
+    The directory used for temporary storage can be changed by specifying the TMPDIR environment variable (on UNIX)
```
Comment (smehringer) [inline manual/source/Tutorial/DataStructures/Indices/StringIndices.rst]: `To circumvent this`, the directory used ...  💅

File: manual/source/Tutorial/DataStructures/Indices/StringIndices.rst (hunk 3 above)
Comment (h-2) [inline manual/source/Tutorial/DataStructures/Indices/StringIndices.rst]: no, no comma there! But it doesn't sound right in any case. I would suggest:
```
When building large indexes, it is therefore possible to run out of disk space (in which case an exception will be thrown).
```

**Pull request merged**